src/libraries/System.Private.CoreLib/src/System/Reflection/InvokeUtils.cs

```diff
@@ -123,5 +123,181 @@ private static bool TryConvertPointer(object srcObject, [NotNullWhen(true)] out
             dstPtr = null;
             return false;
         }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
I don't think `AggressiveInlining` is appropriate for a function this big. I suggest creating some microbenchmarks to measure if this is actually appropriate.

Reply:
https://github.com/dotnet/runtime/pull/102739#discussion_r1636070953

Reply:
When was this measured?

Reply:
Just recent of this PR.

Reply:
> I don't think AggressiveInlining is appropriate for a function this big. 

It is ok when there is a very limited number of callers and some are perf sensitive. I would add a comment like the one that's in the native AOT clone of this method to explain why it is ok: https://github.com/dotnet/runtime/blob/03f58cb2790e9ef59b0d7d0eb60afd843ad103dc/src/coreclr/nativeaot/System.Private.CoreLib/src/System/InvokeUtils.cs#L152

> When was this measured?

Less than two weeks ago as you can see in the comment. 

The alternatives to AggresiveInlining are:
- Duplicate the implementation 
   or 
- Take ~30% performance regression for the type conversion path of Array.Copy

Reply:
Okay.